lib/ecto/adapters/mysql/connection.ex

```diff
@@ -514,7 +514,7 @@ if Code.ensure_loaded?(Mariaex.Connection) do
 
     defp ecto_to_db(:string),     do: "varchar"
     defp ecto_to_db(:float),      do: "double"
-    defp ecto_to_db(:datetime),   do: "datetime"
+    defp ecto_to_db(:datetime),   do: "datetime(6)"
```

Comment:
It would be nice if it was easy to support older versions too. But MySQL 5.6 existed in 2011. It's older than Elixir. Does Ecto need to be held back to support MySQL versions from 2011?

Maybe Ecto should simply require MySQL 5.6+